src/components/measurements/measurementsD3.js

```diff
@@ -492,3 +494,38 @@ export const addColorByAttrToGroupingLabel = (ref, treeStrainColors) => {
       }
     });
 };
+
+const colorGroupingCrosshairId = "measurementsColorGroupingCrosshair";
+export const removeColorGroupingCrosshair = (ref) => {
+  const svg = select(ref);
+  svg.select(`#${colorGroupingCrosshairId}`).remove();
+};
+
+export const addGroupingValueCrosshair = (ref, groupingValue) => {
+  // Remove previous color grouping crosshair
+  removeColorGroupingCrosshair(ref);
+
+  const svg = select(ref);
+  svg.selectAll(`.${classes.yAxis}`).select(".tick")
+    .each((_, i, elements) => {
+      const groupingLabel = select(elements[i]);
+      const currentGroupingValue = groupingLabel.select(`.${classes.groupingValue}`).text()
+      if (groupingValue === currentGroupingValue){
+        const {width} = groupingLabel.node().getBoundingClientRect();
+        groupingLabel.append("svg")
+          .attr("id", colorGroupingCrosshairId)
+          .attr("stroke", "currentColor")
+          .attr("fill", "currentColor")
+          .attr("strokeWidth", "0")
+          .attr("viewBox", "0 0 256 256")
+          .attr("height", layout.yAxisColorByLineHeight * 2)
+          .attr("width", layout.yAxisColorByLineHeight * 2)
+          .attr("x", -width - (layout.yAxisColorByLineHeight * 2))
+          .attr("y", -layout.yAxisColorByLineHeight)
```

Comment:
Hmm, placement of the crosshair is not great when the grouping string is too long
![Screenshot 2025-01-03 at 2 50 12 PM](https://github.com/user-attachments/assets/ec17b506-1c55-4bb1-ab66-eba82ee80edd)

Reply:
Noted this in related issue https://github.com/nextstrain/auspice/issues/1822.